site/layouts/shortcodes/blocks/feature.html

```diff
@@ -0,0 +1,10 @@
+{{ $icon := .Get "icon" | default "fa-lightbulb" }}
```

Comment:
@TOsmanov , did you investigate problems with docsy? Can we avoid adding the file into repo?